### ISSUE TITLE
Dupe logs in Warning view

### ISSUE THREAD

**Issue by duramato** [pymedusa/SickRage#560]
Branch/Commit: Develop
What happened: Logs from backlog showing "dupped" in warning view
What you expected: To only show 1 of the same in warning view
Small snipped from warning logs, with only important dupe logs:
```
2016-05-09 12:36:10 SEARCHQUEUE-BACKLOG-78904 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 12:28:43 SEARCHQUEUE-BACKLOG-287254 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 12:23:34 SEARCHQUEUE-BACKLOG-278798 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 12:08:11 SEARCHQUEUE-BACKLOG-254184 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 12:06:05 SEARCHQUEUE-BACKLOG-254184 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 11:51:25 SEARCHQUEUE-BACKLOG-277096 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 11:28:54 SEARCHQUEUE-BACKLOG-110381 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 10:47:37 SEARCHQUEUE-BACKLOG-82186 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 10:42:26 SEARCHQUEUE-BACKLOG-82186 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 10:26:42 SEARCHQUEUE-BACKLOG-83982 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 10:06:53 SEARCHQUEUE-BACKLOG-100741 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:59:06 SEARCHQUEUE-BACKLOG-100741 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 09:56:27 SEARCHQUEUE-BACKLOG-281341 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:44:10 SEARCHQUEUE-BACKLOG-271546 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:40:27 SEARCHQUEUE-BACKLOG-273859 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:32:42 SEARCHQUEUE-BACKLOG-263546 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:28:50 SEARCHQUEUE-BACKLOG-272876 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:20:04 SEARCHQUEUE-BACKLOG-292136 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 09:13:24 SEARCHQUEUE-BACKLOG-294117 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 08:50:12 SEARCHQUEUE-BACKLOG-268592 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 08:38:27 SEARCHQUEUE-BACKLOG-280473 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 08:07:33 SEARCHQUEUE-BACKLOG-259765 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 08:02:15 SEARCHQUEUE-BACKLOG-77904 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 07:39:48 SEARCHQUEUE-BACKLOG-72514 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
2016-05-09 07:32:17 SEARCHQUEUE-BACKLOG-247897 :: [Demonoid] :: [55c0658] Unable to find episode with date 0005-02-16 for show Australian Story, skipping
2016-05-09 07:16:43 SEARCHQUEUE-BACKLOG-77466 :: [Rarbg] :: [55c0658] Cant find search_tvdb in database. Are you sure it exists?
```

**Comment by OmgImAlexis** [comment]
Aren't they all in different threads since the number after BACKLOG is different for all of them?

**Comment by duramato** [comment]
Yes  but for example  daily does not show the diferente ones just `SEARCHQUEUE-DAILY-SEARCH`,  that's probably what makes it show more than once. 
Even tough the same warning in a backlog is the same in  another backlog,  a time out this backlog  search  is the same as in the next one :P,  only need to be informed  once via  Warning view

**Comment by fernandog** [comment]
Can't find the code the filter by error/warning message.

**Comment by ratoaq2** [comment]
The original issue was fixed some time ago. The other source of dupe logs was because < > characters were not escaped and it was fixed by #890
